src/box/lua/console.lua

```diff
@@ -278,6 +278,16 @@ local function set_language(storage, value)
         local msg = 'Invalid language "%s", supported languages: lua and sql.'
         return error(msg:format(value))
     end
+    if value == 'sql' then
+        local is_configured = false
+        for k, _ in pairs(box) do
+            is_configured = is_configured or k == 'execute'
+        end
+        if not is_configured then
+            local msg = "Unable to set language to 'sql' with unconfigured BOX"
```

Comment:
Nit: I never seen `box` capitalized. It seems, it is unusual naming style for it.

Reply:
Thank you, fixed.

Reply:
That's minor, but I would also update it in the changelog entry.